README.md

```diff
@@ -1,13 +1,19 @@
+<div align="center">
+
 # Halyard
 
-[![Build Status](https://api.travis-ci.org/spinnaker/halyard.svg?branch=master)](https://travis-ci.org/spinnaker/halyard)
+<a href="https://app.travis-ci.com/spinnaker/halyard">
```

Comment:
Definitely appreciate improvements here.  Turns out https://app.travis-ci.com/spinnaker/halyard is a dead link.  We build with github actions these days.  I'm all for including the logo though.  Mind doing it on its own, without the travis link?

Reply:
ok no problem